### PR TITLE
[substrate] INFRA-47 updated examples

### PR DESCRIPTION
**Pull request by sownak**
Signed-off-by: sownak <sownak.roy@accenture.com>

**Changelog**
- Update charts for api and frontend
- Fix ipfsbootnode files

### CODE REVIEW

File: platforms/substrate/charts/substrate-key-mgmt/templates/job.yaml [hunk 1]
```diff
@@ -167,27 +167,35 @@ spec:
...
               {
                 \"data\": {
                 \"node_id\": \"$node_id\",
                 \"node_key\": \"$(cat node_key)\",
-                \"aura_seed\": \"$ACC_SEED\",
-                \"aura_addr\": \"$ACC_ADDR\",
+                \"aura_seed\": \"$AURA_SEED\",
+                \"aura_addr\": \"$AURA_ADDR\",
                 \"grandpa_seed\": \"$GRANDPA_SEED\",
                 \"grandpa_addr\": \"$GRANDPA_ADDR\",
```
Comment (kaverikhaneja) [inline platforms/substrate/charts/substrate-key-mgmt/templates/job.yaml:194]: Do we need grandpa and aura keys for member nodes?

File: platforms/substrate/charts/vitalam-ipfs-node/templates/configmap.yaml [hunk 2]
```diff
@@ -22,4 +22,3 @@ data:
   ipfsApiPort: {{ include "vitalam-ipfs.ipfsApiPort" . }}
   ipfsSwarmPort: {{ .Values.config.ipfsSwarmPort | quote }}
   dataVolumeSize: {{ .Values.storage.dataVolumeSize | quote }}
-
```
Comment (kaverikhaneja) [inline platforms/substrate/charts/vitalam-ipfs-node/templates/configmap.yaml:25]: newline please.

File: platforms/substrate/charts/substrate-key-mgmt/templates/job.yaml (hunk 1 above)
Comment (sownak) [reply platforms/substrate/charts/substrate-key-mgmt/templates/job.yaml:194]: I have kept them to be same as what DC did. Additional account keys are generated for member nodes, thats all.

**Review by kaverikhaneja — APPROVED** [review]

File: platforms/substrate/charts/vitalam-ipfs-node/templates/configmap.yaml (hunk 2 above)
Comment (sownak) [reply platforms/substrate/charts/vitalam-ipfs-node/templates/configmap.yaml:25]: This is fine, the wrong newline was removed

**Pull request merged**